examples/scripts/reinforcement_learning/simple_cart_pole/cart_pole_env.py

```diff
@@ -0,0 +1,122 @@
+
+import os
+import gymnasium as gym
+import numpy as np
+
+from gz.common6 import set_verbosity
+from gz.sim10 import TestFixture, World, world_entity, Model, Link, get_install_prefix
+from gz.math8 import Vector3d
+from gz.transport14 import Node
```

Comment:
I merged this branch with `main` so we now have to remove the major version no. from sim and transport

```suggestion
from gz.sim import TestFixture, World, world_entity, Model, Link, get_install_prefix
from gz.math8 import Vector3d
from gz.transport import Node
```